rdf-delta-server-local/src/main/java/org/seaborne/delta/server/local/patchstores/zk/PatchLogIndexZk.java

```diff
@@ -494,51 +372,50 @@ private void writeLockState(Id session, long ticks) {
             builder.pair(jLockId, session.asPlainString());
             builder.pair(jTicks, ticks);
         });
-        Zk.zkSetJson(client, lockStatePath, value);
+        this.client.setZNode(lockStatePath, value);
     }
 
     @Override
     public LockState readLock() {
-        JsonObject value = Zk.zkFetchJson(client, lockStatePath);
+        JsonObject value = this.client.fetchJson(lockStatePath);
         if ( value == null || value.isEmpty() )
             return LockState.UNLOCKED;
-        String dt = value.getString(jTimestamp);
         // Validate?
         String lockId = value.getString(jLockId);
-        if ( dt == null || lockId == null ) {} // XXX
         long ticks = value.get(jTicks).getAsNumber().value().longValue();
-        LockState lockState = LockState.create(Id.fromString(lockId), ticks);
-        return lockState;
+        return LockState.create(Id.fromString(lockId), ticks);
     }
 
     @Override
     public Id grabLock(Id oldSession) {
```

Comment:
The following block did not use the runWithLock method on this class for some reason. I've chosen to force all such instances through that method to ensure consistent behavior.